### PR TITLE
Exams by Mykola Kushnir IM-22

### PR DESCRIPTION
**Pull request by kushnirko**
I refactored **except.js** and **sum.js**. Make please code review to imrove my solutions.

### CODE REVIEW

**Review by yatsenkoM — APPROVED** [review]
I like this code. I did not find any problems during the review. Very good job!

**Review by viktoriiakapys — COMMENTED** [review]
Your code is shorter than the previous one and after your changes it became readable. The variable names are clear for perception. Also you simplified the construction of many nested 'if' in the 'sum' file, which is good. In the 'ecxept' file you rewrote a long algorithm to a more understandable and more optimized one. I liked the work you did and the well-written code!

**Review by viktoriiakapys — APPROVED** [review]

File: Tasks/sum.js [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
-  for (i of s) {
-    let t = typeof i;
-    if (t === 'number') {
-      if (sum.length > 0) {
-        const new_Sum = sum[sum.length - 1] + i;
-        sum.push(new_Sum);
-      }
+'use strict';
+
+const sumNumsFromArr = (arr) => {
```
Comment (tshemsedinov) [inline Tasks/sum.js:5]:
```suggestion
const sumNumbers = (arr) => {
```

**Comment by tshemsedinov** [comment]
LGTM (looks good to me)

**Pull request closed without merging**